man/flac.md

```diff
@@ -220,45 +223,29 @@ the HTML documentation.
 **-c, \--stdout**
 :	Write output to stdout
 
-**-s, \--silent**
-:	Silent mode (do not write runtime encode/decode statistics to stderr)
-
-**\--totally-silent**
-: Do not print anything of any kind, including warnings or errors. The
-	exit code will be the only way to determine successful completion.
-
-**\--no-utf8-convert**
-:	Do not convert tags from local charset to UTF-8. This is useful for
-	scripts, and setting tags in situations where the locale is wrong.
-	This option must appear before any tag options!
-
-**-w, \--warnings-as-errors**
-:	Treat all warnings as errors (which cause flac to terminate with a
-	non-zero exit code).
-
 **-f, \--force**
 :	Force overwriting of output files. By default, flac warns that the
 	output file already exists and continues to the next file.
 
-**-o** *filename***, \--output-name=***filename*
+**\--delete-input-file**
+:	Automatically delete the input file after a successful encode or
+	decode. If there was an error (including a verify error) the input
+	file is left intact.
+
+**-o** _filename_, **\--output-name=** _filename_
 :	Force the output file name (usually flac just changes the extension).
 	May only be used when encoding a single file. May not be used in
 	conjunction with \--output-prefix.
 
-**\--output-prefix=***string*
+**\--output-prefix=** _string_
 :	Prefix each output file name with the given string. This can be
 	useful for encoding or decoding files to a different directory. Make
 	sure if your string is a path name that it ends with a trailing \`/'
 	(slash).
 
-**\--delete-input-file**
-:	Automatically delete the input file after a successful encode or
-	decode. If there was an error (including a verify error) the input
-	file is left intact.
-
 **\--preserve-modtime**
-:	Output files have their timestamps/permissions set to match those of
-	their inputs (this is default). Use \--no-preserve-modtime to make
+:	(default!) Output files have their timestamps/permissions set to 
```

Comment:
Was this change intentional or were you planning on improving this later? I think '(default!)' is somewhat ambiguous.